src/components/map/MapLibre.tsx

```diff
@@ -368,7 +390,7 @@ export const MapLibre = ({
         )}
       </MapView>
 
-      {isMyLocationButtonVisible && showsUserLocation && (
+      {isMyLocationButtonVisible && showsUserLocation && !onMapPress && (
```

Comment:
Why did you add the onMapPress control here? I think we should give the user the ability to choose their own location as well as the `onMapPress` feature.